src/builder.js

```diff
@@ -124,6 +124,23 @@ export default function builder() {
     return false;
   }
 
+  function objectIsEmpty(obj) {
+    return ((function() {
+      var k, results;
+      results = [];
+      for (k in obj) {
+        if (!Object.prototype.hasOwnProperty.call(obj, k)) continue;
```

Comment:
I know you just refactored this as part of the PR, but this is doing unneeded work. You could just check for hasOwnProperty and return true once it is found, instead of collecting all the keys for a count at the end.

Reply:
@rabidpraxis thanks! I think this does the trick: https://github.com/honeybadger-io/honeybadger-js/pull/154/commits/46d8fab779a13ca052b9d316c7e3903ee3d1664e